### DISCUSSION TITLE
cronjob as documented no longer works for docker image

### DISCUSSION THREAD

**Discussion by neiaberau** [kanboard/kanboard#5290] [General]
cronjob as documented here 

  https://docs.kanboard.org/v1/admin/cronjob/
with example
  0 8 * * * cd /path/to/kanboard && ./cli cronjob >/dev/null 2>&1

no longer works with newest docker image, I get for (paths adapted and working until recent image update)
         cd /var/www/app/cli && ./cli cronjob

env: can't execute 'php': No such file or directory

find / -name php 
   finds no php executable
however there is php82 executable, symlinking works

**Comment by fguillot** [comment]
You can use the `php82` command to run it: `php82 ./cli cronjob`.

For reference, here is the crontab used in actual the Docker image: https://github.com/kanboard/kanboard/blob/e182856b4b25d589af0a5945a85cdae8bd44fca1/docker/etc/crontabs/nginx#L1

The documentation provides only an example that might need to be adapted depending how you installed the software.

**Reply by neiaberau** [reply]
yes, but I call it from outside the container as docmented, which does not work, 
the reason is:

the image changed


the old image had: fd14805f47a3 

/usr/bin # ls -l php*
lrwxrwxrwx    1 root     root             5 Apr  8 21:13 php -> php81
-rwxr-xr-x    1 root     root       8399200 Mar 21 00:20 php81


the new image: a30820391f45
/ # cd /usr/bin
/usr/bin # ls -l php*
-rwxr-xr-x    1 root     root       8394624 May 10 05:42 php82


the symlink is missing in the new image
